frontend/src/pages/ActivityReport/Pages/components/GoalPicker.js

```diff
@@ -193,7 +193,7 @@ const GoalPicker = ({
         >
           Keep objective
         </Button>
-        <Button type="button" onClick={onRemove} unstyled>Remove objective</Button>
+        <Button type="button" onClick={onRemove} className="usa-button--unstyled">Remove objective</Button>
```

Comment:
Should this use the subtle variant?

Reply:
Yep! 686bcfa536020805787e01397cde4eb1e1163d35